pkgs/data/themes/cyberpunk-neon/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "cyberpunk-neon";
+  version = "2021-02-02";
+
+  src = fetchFromGitHub {
+    owner = "Roboron3042";
+    repo = "Cyberpunk-Neon";
+    rev = version;
```

Comment:
Releases got removed upstream. Can you fetch a commit instead and change the version number to unstable-XXXX-XX-XX?

Reply:
Done, thanks.